index.js

```diff
@@ -460,7 +460,24 @@ NYC.prototype.checkCoverage = function (thresholds) {
 }
 
 NYC.prototype._loadProcessInfoTree = function () {
-  return ProcessInfo.buildProcessTree(this._loadProcessInfos())
+  var _this = this
+  var processTree = ProcessInfo.buildProcessTree(this._loadProcessInfos())
```

Comment:
I'd be tempted to encapsulate this logic inside `process.js`; and all that `index.js` would need to do would be:

`console.log(processInfo.render())`.